sql/core/src/test/scala/org/apache/spark/sql/execution/command/AlterTableAddPartitionSuiteBase.scala

```diff
@@ -22,6 +22,20 @@ import org.apache.spark.sql.catalyst.analysis.PartitionsAlreadyExistException
 import org.apache.spark.sql.catalyst.catalog.CatalogTypes.TablePartitionSpec
 import org.apache.spark.sql.internal.SQLConf
 
+/**
+ * This base suite contains unified tests for the `ALTER TABLE .. ADD PARTITION` command that
+ * check V1 and V2 table catalogs. The tests that cannot run for all supported catalogs are
+ * located in more specific test suites:
+ *
+ *   - V2 table catalog tests:
+ *     [[org.apache.spark.sql.execution.command.v2.AlterTableAddPartitionSuite]]
+ *   - V1 table catalog tests:
+ *     [[org.apache.spark.sql.execution.command.v1.AlterTableAddPartitionSuiteBase]]
```

Comment:
`org.apache.spark.sql.execution.command.AlterTableAddPartitionSuiteBase`?

Reply:
You pointed out the current trait which contains tests for v1 and v2. `org.apache.spark.sql.execution.command.v1.AlterTableAddPartitionSuiteBase` is the base trait for v1 In-Memory and Hive catalogs.